database/profiles.go

```diff
@@ -0,0 +1,54 @@
+package database
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/forbole/bdjuno/v2/types"
+)
+
+// SaveProfilesParams save the params of profiles module in the database
+func (db *Db) SaveProfilesParams(params *types.ProfilesParams) error {
+	nicknameParamsBz, err := json.Marshal(&params.NicknameParams)
+	if err != nil {
+		return fmt.Errorf("error while marshaling Nickname params: %s", err)
+	}
+	dTagParamsBz, err := json.Marshal(&params.DTagParams)
+	if err != nil {
+		return fmt.Errorf("error while marshaling DTag params: %s", err)
+	}
+	bioParamsBz, err := json.Marshal(&params.BioParams)
+	if err != nil {
+		return fmt.Errorf("error while marshaling Bio params: %s", err)
+	}
+	oracleParamsBz, err := json.Marshal(&params.OracleParams)
+	if err != nil {
+		return fmt.Errorf("error while marshaling Oracle params: %s", err)
+	}
+
+	stmt := `
+INSERT INTO profiles_params (nickname_params, d_tag_params, bio_params, oracle_params, height) 
```

Comment:
Instead of storing the various params separately, can we store them the same way we do with other modules? See the [staking params](https://github.com/forbole/bdjuno/blob/v2/cosmos/stargate/database/staking_params.go#L21), the [slashing params](https://github.com/forbole/bdjuno/blob/v2/cosmos/stargate/database/slashing.go#L60), the [gov params](https://github.com/forbole/bdjuno/blob/v2/cosmos/stargate/database/gov.go#L37), and the [distribution params](https://github.com/forbole/bdjuno/blob/v2/cosmos/stargate/database/distribution.go#L42)

Reply:
sure @RiccardoM, I have updated the codes. However the linter and the test bot are showing "cannot find module providing package github.com/desmos-labs/desmos/app"  I was digging but not sure where went wrong.  do you have any idea.. ?